routes/web.php

```diff
@@ -94,6 +94,8 @@
 Route::post('/api/Diagnostics',                 'DiagnosticsController@show');
 Route::get('/Diagnostics',                      'DiagnosticsController@show');
 
+Route::get('/api/Update',                       'UpdateController@do')->middleware('admin');
```

Comment:
So how will this be invoked? An admin-only menu option in the front end? If you have that code ready, can you push it so that I can actually try using this? :smile:

Reply:
Given that it is a get request you could also directly access it via https://example.com/api/Update :)
But as you suggested, I added a button in the front-end.